cmake/HunterGate.cmake

```diff
@@ -0,0 +1,540 @@
+# Copyright (c) 2013-2018, Ruslan Baratov
```

Comment:
it this copyright statement intended to be different from the rest of the files?